src/api/routes/policies/stats.ts

```diff
@@ -0,0 +1,18 @@
+import { Router, Request, Response } from "express";
+import { route } from "@fosscord/api";
+import { Config, User, UserPublic, Member, Guild, Message } from "@fosscord/util";
+const router = Router();
+
+
+router.get("/",route({}), async (req: Request, res: Response) => {
+    let users,guilds,msgs,memberships;
+	// needs to be let otherwise we can't for
+
+	users = await User.count();
+	guilds = await Guild.count();
+	msgs = await Message.count();
+	memberships = await Member.count();
```

Comment:
why not just do
```ts
let users = await User.count();
let guilds = await Guild.count();
...
```

Reply:
in a future commit, i will add a parametrisation to filter out some of them
hence i left that possibility